src/plans/alm_plan.jl

```diff
@@ -186,7 +186,7 @@ end
 # Allocating vector -> we can omit a few of the ineq gradients.
 function (
     LG::AugmentedLagrangianGrad{
-        <:ConstrainedProblem{<:AllocatingEvaluation,<:VectorConstraint}
+        <:ConstrainedProblem{AllocatingEvaluation,<:VectorConstraint}
```

Comment:
Oh I am never sure about the `<:`such that here I thought – there is no harm in having that ;)